TheOneScriptingEngine/PlayerScript.cs

```diff
@@ -11,6 +11,10 @@ public override void Update()
 
 		//Keyboard
 
+		if (Input.GetKeyboardButton(Input.KeyboardCode.K)) {
+			SceneManager.LoadScene("Scene");
```

Comment:
"(Input.GetKeyboardButton(Input.KeyboardCode.K)" -> This should be erased, K is only used to debug